pkg/api/job_spec.go

```diff
@@ -111,3 +112,11 @@ func ResolveSpecFromEnv() (*JobSpec, error) {
 
 	return spec, nil
 }
+
+func (s *JobSpec) GetJobSpecUpstream() (*downwardapi.JobSpec, error) {
+	spec := &downwardapi.JobSpec{}
+	if err := json.Unmarshal([]byte(s.RawSpec()), spec); err != nil {
+		return nil, err
+	}
+	return spec, nil
```

Comment:
I think the check is redundant, just return `spec, err`?

Reply:
Removed.